native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -0,0 +1,151 @@
+import org.ballerinalang.formatter.core.Formatter;
+import org.ballerinalang.formatter.core.FormatterException;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Class to implement "persist init" command for ballerina.
+ */
+
+@CommandLine.Command(
+        name = "init",
+        description = "generate database configurations.")
+
+public class Init implements BLauncherCmd {
+    private final PrintStream outStream = System.out;
+    private final PrintStream errStream = System.err;
+    private String sourcePath = "";
+    private ProjectEnvironmentBuilder projectEnvironmentBuilder;
+    private String configPath = PersistToolsConstants.CONFIG_PATH;
+
+    private CommandLine parentCmdParser;
+    private String name = "";
+
+    Project balProject;
+
+    @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
+    private boolean helpFlag;
+
+    @Override
+    public void execute() {
+        try  {
+            if (projectEnvironmentBuilder == null) {
+                Project balProject = ProjectLoader.loadProject(Paths.get(""));
+                String orgName = balProject.currentPackage().descriptor().org().value();
+                String projectName = balProject.currentPackage().descriptor().name().value();
+                name = orgName + "." + projectName;
+
+            } else {
+                Project balProject = ProjectLoader.loadProject(Paths.get(this.sourcePath), projectEnvironmentBuilder);
+                String orgName = balProject.currentPackage().descriptor().org().value();
+                String projectName = balProject.currentPackage().descriptor().name().value();
+                name = orgName + "." + projectName;
+            }
+        } catch (ProjectException e) {
+            errStream.println("The current directory is not a Ballerina project!");
+            return;
+        }
+        if (!Files.exists(Paths.get(sourcePath, configPath))) {
+            try {
+                createConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when creating the Config.toml file: " + e.getMessage());
+            }
+        } else {
+            try {
+                updateConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when updating the Config.toml file: " + e.getMessage());
+            }
+        }
+    }
+    private void createConfigToml() throws Exception {
+        SyntaxTree toml = CreateSyntaxTree.createToml();
+        writeOutputFile(toml, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void updateConfigToml() throws Exception {
+        SyntaxTree toml = CreateSyntaxTree.updateToml(Paths.get(this.sourcePath, this.configPath), this.name);
```

Comment:
```suggestion
        SyntaxTree syntaxTree = CreateSyntaxTree.updateToml(Paths.get(this.sourcePath, this.configPath), this.name);
```

Reply:
Updated in [7ad69ea](https://github.com/ballerina-platform/persist-tools/pull/9/commits/7ad69eacc4693716890747ef60b3859f0ad5b2b1)